app/components/watch-show.js

```diff
@@ -11,28 +12,54 @@ export default Component.extend(
 
     tagName: '',
 
-    url: oneWay('wikiVariables.watchShowURL'),
     buttonLabel: oneWay('wikiVariables.watchShowButtonLabel'),
+    imageUrl: oneWay('wikiVariables.watchShowImageURL'),
+    cta: oneWay('wikiVariables.watchShowCTA'),
+    trackingPixelURL: oneWay('wikiVariables.watchShowTrackingPixelURL'),
 
-    isVisible: computed('url', function () {
+    url: computed('wikiVariables.watchShowURL', 'wikiVariables.watchShowURLIOS', 'wikiVariables.watchShowURLAndroid', function () {
+      if (this.wikiVariables.watchShowURL) {
+        return this.wikiVariables.watchShowURL;
+      } else if (system === 'ios') {
+        return this.wikiVariables.watchShowURLIOS;
+      } else {
+        return this.wikiVariables.watchShowURLAndroid;
+      }
+    }),
+
+    isVisible: computed('url', 'buttonLabel', 'geo', function () {
       return this.url && this.buttonLabel && this.geo.country === 'US';
     }),
 
     didInsertElement() {
       this._super(...arguments);
 
+      if (!this.isVisible) {
```

Comment:
I found an issue that we are tracking watch show impression even if the button is not visible :)